ansible/site.yml

```diff
@@ -22,97 +22,55 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 #!/usr/bin/env ansible-playbook
-# Top-level site.yml for User Access Nodes
-- hosts: Application_UAN
-  gather_facts: yes
+---
+- name: setup cfs_image host group
+  gather_facts: false
+  hosts: all
+  tasks:
+    - name: Add image customization hosts to the cfs_image host group.
+      add_host:
+        name: '{{ inventory_hostname }}'
+        groups: cfs_image
+      when: cray_cfs_image
+
+- hosts: Application_UAN:&cfs_image
+  gather_facts: true
   any_errors_fatal: true
   remote_user: root
 
-  pre_tasks:
```

Comment:
The setting and unsetting of /etc/nologin should be in their own "site.yml" files so they can be layered at the beginning and the end of the UAN CFS process.  We don't want regular users attempting to access and use the UAN until the final layer has completed.  So the first and last layer should be something like "uan_set_nologin.yml" and "uan_unset_nologin.yml" in the UAN CFS configuration.

Reply:
Good idea, I added the two new playbooks.